starter-code/views/index.ejs

```diff
@@ -1,11 +1,6 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title><%= title %></title>
-    <link rel='stylesheet' href='/stylesheets/style.css' />
-  </head>
-  <body>
-    <h1><%= title %></h1>
-    <p>Welcome to <%= title %></p>
-  </body>
-</html>
+<h1><%= title %></h1>
+<p>Welcome to <%= title %></p>
+
+<p><a href="/signup">Signup Here</a></p>
+<p><a href="/login">Login Here</a></p>
```

Comment:
If I'm already login, I will see this links. Maybe we can do something for the user that is already login can't see this links.